try {
  a();
} catch (e) {
  console.log(e);
}
